packages/waku/src/lib/middleware/rsc/worker-impl.ts

```diff
@@ -234,7 +226,11 @@ const resolveClientEntry = (
       );
     }
   }
-  return config.basePath + path.relative(root, filePath);
+  return (
+    (command === 'dev' ? '/@id' : '') +
```

Comment:
What's `@id`? It must be different from `@fs`.

Reply:
[Yes, they're so different](https://github.com/vitejs/vite/blob/main/packages/vite/src/node/constants.ts#L57C1-L65)

Reply:
Interesting. I might be missing something, but what are not valid browser import specifiers? Any example in our case? I mean, we didn't need this before.

Reply:
Yes, when we changed roots, vite tried to identify /src/components/Coutner.tsx in its own way which caused it to create some virtual modules that load the index.html css styles in those virtual modules and when we loaded them in the server we faced the error that I shared above.  
https://github.com/dai-shi/waku/pull/181#discussion_r1405399116

https://github.com/vitejs/vite/blob/2fce647edae3c93915f43c2a2971b87aa2db53c3/packages/vite/src/node/server/middlewares/indexHtml.ts#L178C2-L191

https://github.com/vitejs/vite/blob/2fce647edae3c93915f43c2a2971b87aa2db53c3/packages/vite/src/node/server/middlewares/indexHtml.ts#L333C1-L358


I just added /@id/ and it fixed the issue which led vite to skip this process.

Reply:
Ah, because we changed root.
Please add the source comment too.

I was hoping to make DEV and PRD as similar as possible in the code base, but in the future, we may revisit the design anyway.